frontend/src/features.ts

```diff
@@ -7,6 +7,7 @@ export interface Feature {
 export enum FeatureKey {
   V2 = 'v2', // Please start using V2_ALPHA instead of V2, because we have switched to V2_ALPHA as V2 feature is enabled by default.
   V2_ALPHA = 'v2_alpha',
+  FUNCTIONAL = 'functional',
```

Comment:
`functional` is a common term that refers to many things. Suggest to use `functional_component` as the key name. Also suggest to add a comment to explain where is this flag being expected to use.